charts/metrics-server/templates/certificate.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if eq .Values.tls.type "cert-manager" }}
+  {{- if not .Values.tls.certManager.existingIssuer.enabled }}
```

Comment:
For consistency this shouldn't be indented (or the end block).

Reply:
Consitency across other files in this repo, right?

My intent was to highlight that this IF block does only cover Line 2..12 (the `kind: Issuer` object)

Reply:
Helm indentation is hard, but the idiom in this repo is to not indent beyond the content.

Reply:
Will update the PR

Reply:
done